### PR TITLE
Feature-IOPS-1513-Profiling England Provenance

### PR DESCRIPTION
**Pull request by KazeemHamzat**
Profiling England Provenance

### CODE REVIEW

**Review by digitalhealthmartin — DISMISSED** [review]
Identified no issues.

File: StructureDefinition/England-Provenance.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    </element>
+    <element id="Provenance.agent">
+      <path value="Provenance.agent" />
+      <mustSupport value="true" />
+    </element>
+    <element id="Provenance.agent.who">
+      <path value="Provenance.agent.who" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-PractitionerRole" />
```
Comment (APageNHS) [inline StructureDefinition/England-Provenance.xml:81]: As per discussion, should not be referencing profiles

File: StructureDefinition/England-Provenance.xml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    </element>
+    <element id="Provenance.signature.when">
+      <path value="Provenance.signature.when" />
+      <mustSupport value="true" />
+    </element>
+    <element id="Provenance.signature.who">
+      <path value="Provenance.signature.who" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner" />
```
Comment (APageNHS) [inline StructureDefinition/England-Provenance.xml:115]: As per discussion about references

File: StructureDefinition/England-Provenance.xml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    </element>
+    <element id="Provenance.signature.who.reference">
+      <path value="Provenance.signature.who.reference" />
+      <min value="1" />
+    </element>
+    <element id="Provenance.signature.onBehalfOf.identifier.assigner">
+      <path value="Provenance.signature.onBehalfOf.identifier.assigner" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
```
Comment (APageNHS) [inline StructureDefinition/England-Provenance.xml:131]: As per discussion about references

File: StructureDefinition/Extension-England-ProvenanceAgent.xml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="https://fhir.nhs.uk/StructureDefinition/Extension-England-ProvenanceAgent" />
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]" />
+      <min value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-PractitionerRole" />
```
Comment (APageNHS) [inline StructureDefinition/Extension-England-ProvenanceAgent.xml:59]: As per discussion about references to ukcore

File: StructureDefinition/England-Provenance.xml
Comment (digitalhealthmartin) [inline StructureDefinition/England-Provenance.xml:1]: Missing `<date value="2023-06-01" />`

File: StructureDefinition/England-Provenance.xml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the FHIR UK Core resource [UKCore Provenance](https://simplifier.net/hl7fhirukcorer4/ukcore-provenance)." />
```
Comment (digitalhealthmartin) [inline StructureDefinition/England-Provenance.xml:19]: UKCore -> UK Core

File: StructureDefinition/Extension-England-ProvenanceAgent.xml [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-England-ProvenanceAgent.xml:60]: All targetProfiles should be removed

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]
Noticed few things to be addressed

File: StructureDefinition/Extension-England-ProvenanceAgent.xml [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+    </element>
+    <element id="Extension.value[x].identifier.value">
+      <path value="Extension.value[x].identifier.value" />
+      <min value="1" />
+    </element>
+    <element id="Extension.value[x].identifier.assigner">
+      <path value="Extension.value[x].identifier.assigner" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-England-ProvenanceAgent.xml:125]: targetProfile should be removed

File: StructureDefinition/Extension-England-ProvenanceAgent.xml [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+      </type>
+    </element>
+    <element id="Extension.value[x].identifier.assigner.identifier.assigner">
+      <path value="Extension.value[x].identifier.assigner.identifier.assigner" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-England-ProvenanceAgent.xml:132]: targetProfile should be removed

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]
Previous issues resolved. But still a few targetProfiles to remove

**Review by digitalhealthmartin — DISMISSED** [review]
Fixed done. Approved.

**Comment by SheenaSharma** [comment]
Approved.

**Review by SheenaSharma — APPROVED** [review]

**Review by SheenaSharma — CHANGES_REQUESTED** [review]
Missing Purpose in Extension-England-ProvenanceAgent.xml.

File: StructureDefinition/Extension-England-ProvenanceAgent.xml
Comment (SheenaSharma) [inline StructureDefinition/Extension-England-ProvenanceAgent.xml:1]: Missing Purpose.

File: CapabilityStatement/England-CapabilityStatement.xml
Comment (digitalhealthmartin) [inline CapabilityStatement/England-CapabilityStatement.xml:1]: CS includes resources that are not used in this feature branch. Should just be Provenance in this CS

**Review by digitalhealthmartin — DISMISSED** [review]
Review CS

**Review by SBajanki1 — DISMISSED** [review]

**Review by SheenaSharma — DISMISSED** [review]

**Review by SBajanki1 — APPROVED** [review]

**Review by SheenaSharma — APPROVED** [review]

**Pull request merged**